director/lib/conversion_types.py

```diff
@@ -18,7 +18,17 @@ class UnknownMimeTypeError(ValueError):
 class ConversionFormat(typing.NamedTuple):
     format_id: str
     mimetypes: typing.Iterable[str]
-    output_extension: typing.Optional[str] = None
+    extensions: typing.List[str] = []
+
+    @property
+    def default_extension(self) -> str:
+        """Get the default extension for the format."""
+        return self.extensions[0] if len(self.extensions) else self.format_id
+
+    @property
+    def is_binary(self) -> bool:
+        """Is the format be considered binary when determining the type of HTTP response."""
+        return self.format_id not in ("html", "json", "jsonld", "md", "rmd", "xml")
 
 
 class ConversionFormatId(enum.Enum):
```

Comment:
Can you put a comment in this class that the format_ids must be lower case. Hopefully it's obvious but seeing the use in `is_binary` above made me think we should point it out explicitly.